cibuildwheel/docker_container.py

```diff
@@ -100,9 +100,11 @@ def __exit__(
         exc_tb: Optional[TracebackType],
     ) -> None:
 
+        self.bash_stdin.write(b"exit 0\n")
+        self.bash_stdin.flush()
+        self.process.wait(timeout=5)
```

Comment:
Should we fail on timeout or just `docker kill` the container and try to wait again (and go on) ?

Reply:
Failing here would be a non-critical error, so it would be acceptable to just log a warning and move on. But until we see such an error reported to us, I don't think it's necessary to code around it - this seems okay to me.

But, maybe it would be good to increase this timeout to decrease the chance of this happening, on a remote container for example. It would be frustrating to fail a build because of this. And if we do ever get an error report, we'd like to be sure that it really had stallled.

```suggestion
        self.process.wait(timeout=30)
```